src/darwin/Framework/CHIP/MTRDeviceControllerDelegateBridge.mm

```diff
@@ -36,43 +36,43 @@
     }
 }
 
-MTRPairingStatus MTRDevicePairingDelegateBridge::MapStatus(chip::Controller::DevicePairingDelegate::Status status)
+MTRCommissioningStatus MTRDeviceControllerDelegateBridge::MapStatus(chip::Controller::DevicePairingDelegate::Status status)
 {
-    MTRPairingStatus rv = MTRPairingStatusUnknown;
+    MTRCommissioningStatus rv = MTRCommissioningStatusUnknown;
     switch (status) {
     case chip::Controller::DevicePairingDelegate::Status::SecurePairingSuccess:
-        rv = MTRPairingStatusSuccess;
+        rv = MTRCommissioningStatusSuccess;
         break;
     case chip::Controller::DevicePairingDelegate::Status::SecurePairingFailed:
-        rv = MTRPairingStatusFailed;
+        rv = MTRCommissioningStatusFailed;
         break;
     case chip::Controller::DevicePairingDelegate::Status::SecurePairingDiscoveringMoreDevices:
-        rv = MTRPairingStatusDiscoveringMoreDevices;
+        rv = MTRCommissioningStatusDiscoveringMoreDevices;
         break;
     }
     return rv;
 }
 
-void MTRDevicePairingDelegateBridge::OnStatusUpdate(chip::Controller::DevicePairingDelegate::Status status)
+void MTRDeviceControllerDelegateBridge::OnStatusUpdate(chip::Controller::DevicePairingDelegate::Status status)
 {
-    NSLog(@"DevicePairingDelegate status updated: %d", status);
+    NSLog(@"DeviceControllerDelegate status updated: %d", status);
 
-    id<MTRDevicePairingDelegate> strongDelegate = mDelegate;
+    id<MTRDeviceControllerDelegate> strongDelegate = mDelegate;
```

Comment:
This is existing code, but give `mDelegate` isn't (currently) weak, calling the local `strongDelegate` is fairly misleading.

Reply:
`mDelegate` is not weak, but could end up going away if our bridge goes away, and dropping its ref.  Is there another naming convention for "keep this thing alive while my block is running"?

Reply:
Oh I see. I guess I would have just called the local `delegate` anyway. I didn't think about the fact that `this` is not managed by ARC here since it's a C++ object.